site/react/App.jsx

```diff
@@ -1,6 +1,6 @@
 import React, { useState, useEffect } from 'react'
 
-import { listFrameworks } from '../../dist'
+import { listFrameworks } from '../../dist/index.js'
```

Comment:
Added a build step to our CI and to the contributing guide so this won't fail on missing import